src/modules/dashboard/components/Extensions/ExtensionActionButton.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useCallback } from 'react';
+import { defineMessages } from 'react-intl';
+import { useRouteMatch, useHistory, useParams } from 'react-router';
+
+import Button, { ActionButton, IconButton } from '~core/Button';
+import { ColonyExtensionQuery } from '~data/index';
+import { ExtensionData } from '~data/staticData/extensionData';
+import { ActionTypes } from '~redux/index';
+import { Address } from '~types/index';
+import { COLONY_EXTENSION_SETUP_ROUTE } from '~routes/index';
+
+const MSG = defineMessages({
+  enable: {
+    id: 'Extensions.ExtensionActionButton.enable',
+    defaultMessage: 'Enable',
+  },
+  install: {
+    id: 'Extensions.ExtensionActionButton.install',
+    defaultMessage: 'Install',
+  },
+});
+
+interface Props {
+  canInstall: boolean;
+  colonyAddress: Address;
+  extension: ExtensionData;
+  installedExtension?: ColonyExtensionQuery['colonyExtension'] | null;
+}
+
+const ExtensionActionButton = ({
+  canInstall,
+  colonyAddress,
+  extension,
+  installedExtension,
+}: Props) => {
+  const history = useHistory();
+  const { colonyName, extensionId } = useParams();
+  const onSetupRoute = useRouteMatch(COLONY_EXTENSION_SETUP_ROUTE);
+
+  const handleEnableButtonClick = useCallback(() => {
+    history.push(`/colony/${colonyName}/extensions/${extensionId}/setup`);
+  }, [colonyName, extensionId, history]);
+
+  if (!canInstall || onSetupRoute) return null;
+
+  if (!installedExtension) {
+    return (
+      <ActionButton
```

Comment:
Was the per-version install of extensions de-prioritized for this release ?

![Screenshot from 2020-11-17 14-37-02](https://user-images.githubusercontent.com/1193222/99391178-77a5b300-28e2-11eb-9c4a-dd71540b2975.png)